src/any_agent/frameworks/any_agent.py

```diff
@@ -23,6 +23,7 @@
     get_instrumenter_by_framework,
 )
 from any_agent.tracing.trace import _is_tracing_supported
+from any_agent.serving import A2AServer
```

Comment:
The serving imports should all be lazy / optional otherwise any code will break when the deps are not installed

Reply:
Ok